src/server/api/routes/router.js

```diff
@@ -26,6 +30,10 @@ router.use("/mail", mailRouter);
 
 router.use("/token", refreshTokenRouter);
 
+
+// /api/vessel-reports
+router.use("/vessel_reports", vesselReportRouter);
```

Comment:
Small, but significant, thing: The url should be `/api/vessel-reports/`

Reply:
Renamed from"/vessel_reports" to /vessel-reports/